pkg/fronius/symo.go

```diff
@@ -13,10 +13,12 @@ const (
 	PowerDataPath = "/solar_api/v1/GetPowerFlowRealtimeData.fcgi"
 	// ArchiveDataPath is the Fronius API URL-path for archive data
 	ArchiveDataPath = "/solar_api/v1/GetArchiveData.cgi?Scope=System&Channel=Voltage_DC_String_1&Channel=Current_DC_String_1&Channel=Voltage_DC_String_2&Channel=Current_DC_String_2&HumanReadable=false"
+	// MeterDataPath is the Fronius API URL-path for Smart Meter real time data
+	MeterDataPath = "/solar_api/v1/GetMeterRealtimeData.cgi"
 )
 
 type (
-	symoPowerFlow struct {
+	SymoPowerFlow struct {
```

Comment:
Why expose this type? It's not meant to be public (i.e. usable by other Go packages), but is merely a struct to parse the JSON

Reply:
ok

Reply:
```suggestion
	symoPowerFlow struct {
```